<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastSatEntry</class>
 <widget class="QMainWindow" name="FastSatEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fast Sat Entry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnExport">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Satellite</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbSatellite">
       <item>
        <property name="text">
         <string>AO-27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AO-91</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AO-92</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SO-50</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date/Time (UTC)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Callsigns</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="tbCallsigns">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter all callsigns you have worked.&lt;/p&gt;&lt;p&gt;Use comma or semicolon as a separator.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>My callsign</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tbMyCallsign">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>My grid locator</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="tbGridLocator"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbMode">
       <item>
        <property name="text">
         <string>CW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RTTY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSB</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDateTimeEdit" name="dateTime"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnNow">
         <property name="text">
          <string>Now</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnNow</sender>
   <signal>clicked(bool)</signal>
   <receiver>FastSatEntry</receiver>
   <slot>NowClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked(bool)</signal>
   <receiver>FastSatEntry</receiver>
   <slot>ExportClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>NowClicked()</slot>
  <slot>ExportClicked()</slot>
 </slots>
</ui>
